### ISSUE TITLE
Upload a new version of a file to a work accessibility audit 

### ISSUE THREAD

**Issue by kellyChex** [samvera/hyrax#3985]
### Descriptive summary

Descriptive summary
QA Testing for Hyrax
Hyrax Version: 3.0.0.pre.rc1
Hyrax Branch: master
Hyrax Revision: 48ed925
Nurax Version
ENV: production
SHA: 40b47b0d71af1b958e016ebe49212c1740683133
Branch: 2019-09-12-10-00
Updated: 12 Sep 2019 15:00:37

In testing for the Hyrax 3.0 release, the following points are actions that were discovered during a FireFox tab test and axe accessibility audit. Note, only certain issues are being reported at this time.

**Certain ARIA roles must contain particular children**
![Screen Shot 2019-09-12 at 8 06 41 PM](https://user-images.githubusercontent.com/9621070/64834922-dcf06c80-d598-11e9-90d3-4980e5da6b58.png)

**id attribute value must be unique**
![Screen Shot 2019-09-12 at 8 07 14 PM](https://user-images.githubusercontent.com/9621070/64834937-ef6aa600-d598-11e9-8ea6-7aaaae6d29bc.png)

![Screen Shot 2019-09-12 at 8 07 38 PM](https://user-images.githubusercontent.com/9621070/64834954-fd202b80-d598-11e9-84e9-490a562851b5.png)

![Screen Shot 2019-09-12 at 8 07 57 PM](https://user-images.githubusercontent.com/9621070/64834964-08735700-d599-11e9-8635-fbac82bdfc87.png)

![Screen Shot 2019-09-12 at 8 08 15 PM](https://user-images.githubusercontent.com/9621070/64834971-132dec00-d599-11e9-96fc-bf005095cb2f.png)

**`<li>` elements must be contained in a `<ul>` or `<ol>`**

![Screen Shot 2019-09-12 at 8 08 42 PM](https://user-images.githubusercontent.com/9621070/64834992-23de6200-d599-11e9-977a-147abd7450be.png)

![Screen Shot 2019-09-12 at 8 08 58 PM](https://user-images.githubusercontent.com/9621070/64835008-36f13200-d599-11e9-89e5-8b04ea34b79e.png)

![Screen Shot 2019-09-12 at 8 09 31 PM](https://user-images.githubusercontent.com/9621070/64835043-41133080-d599-11e9-9a4d-d3f12e4d7273.png)

### Steps to reproduce the axe accessibility audit

1. Install the Axe plugin or addon from Deque (https://addons.mozilla.org/en-US/firefox/addon/axe-devtools/ is the firefox version).
2. Open https://nurax-dev.curationexperts.com/concern/file_sets/bk128b08j/edit?locale=en and click Versions, click on the "Choose File" button under the "Upload New Version" heading and select a new file 
3. Open the inspector (right-click, and select "Inspect element")
4. Click on the 'Axe' tab
5. Click 'Analyze'

### Video of Tab Test 
https://cl.ly/1ae450b6b83c/download/Screen%252520Recording%2525202019-09-12%252520at%25252008.03%252520PM.mov

Unable to tab through Descriptions, Versions, and Permissions in Firefox. 

Unable to tab through anything in Safari. 

Chrome working well.
